config.json

```diff
@@ -192,7 +192,12 @@
                 {
                     "name": "Encryption - Not Supported",
                     "platforms": ["javascript"]
+                },
+                {
+                    "name": "Block SdlRouterService by vehicle type",
```

Comment:
I feel the name is a little confusing. Can we use something more generic like "OEM Exclusive Apps"?